### ISSUE TITLE
Number Pad Digits not Mapping to Number Keys

### ISSUE THREAD

**Issue by wdfinch** [rbreaves/kinto#369]
On Ubuntu 20.04 it seems like the digit keys on the number pad map to other inputs/features rather than just numbers. I would also like to note that I turned off the use number pad as mouse keys option in Ubuntu. I'm wondering if there is some option in kinto to have the number pad map the normal number keys (1 => 1, 2 => 2, etc.) or if this is an issue is not related to kinto at all.

**Comment by rbreaves** [comment]
This on an official mac keyboard or Windows based keyboard? I don't test much against keyboard w/ numpads but I certainly would want that working. It's most likely an issue with xkeysnail, but I can probably make an update to xkeysnail to get that resolved.

**Comment by wdfinch** [comment]
Yes official Mac keyboard with number pad.

**Comment by rbreaves** [comment]
I don't have the wireless model but I do have a usb model of an older style keyboard by them so I will test against that one. Thanks

**Comment by rbreaves** [comment]
After doing some testing it appears like it does actually work for me after thinking the toggle didn't work though.. I wonder if restarting the xkeysnail service had helped.

1st try to just press the clear key on your numpad and if that does not work then restart your kinto service. If still not working then try toggling with the clear key again. That may kick it into gear for you as it did for me.. not really sure why this appears to be an issue, perhaps more of a bug than something not implemented correctly in the key.py file of xkeysnail.

**Comment by wdfinch** [comment]
You're right this fixes it. Maybe a note on this could be included in the docs until it can be fixed.

**Comment by rbreaves** [comment]
I will keep this open for now, and revisit this later. I may make a ticket on xkeysnail as well so the creator and that community can be made aware. Whether I fix it or someone else does it won't really matter, I will upstream the fix into my fork and this project.

**Comment by joshgoebel** [comment]
If this issue could be reproduced on my fork I'd be happy to resolve it.  Sounds very strange.  (perhaps something to do with numlock status?)
